hdf5-vol-daos.rpmlintrc

```diff
@@ -0,0 +1,4 @@
+# This line is mandatory to access the configuration functions
+from Config import *
+
+addFilter('no-binary')
```

Comment:
This would be better if more qualified:

```suggestion
addFilter('hdf5-vol-daos.x86_64: E: no-binary')
```

So as not to paper over future issues in other packages.  No need to push a new patch to fix this, but if you can keep it in mind for your next update on this PR that be great.